packages/trpc/server/routers/viewer/organizations/createTeams.handler.ts

```diff
@@ -51,11 +77,16 @@ export const createTeamsHandler = async ({ ctx, input }: CreateTeamsOptions) =>
     .flatMap((ts) => ts.slug ?? [])
     .concat(userSlugs.flatMap((us) => us.username ?? []));
 
-  const duplicatedSlugs = existingSlugs.filter((slug) => teamNames.includes(slug));
+  const duplicatedSlugs = existingSlugs.filter((slug) =>
+    teamNames.map((item) => slugify(item)).includes(slug)
```

Comment:
was validating ["Teams"] with ["teams'] so this validation wasn't case sensitive